testing/smoke/basic_upgrade/standalone-major-managed.sh

```diff
@@ -26,5 +26,5 @@ data_stream_assert_events ${NEXT_MAJOR_LATEST}
 upgrade_managed ${NEXT_MAJOR_LATEST}
 healthcheck 1
 send_events
-Assert there are 2 instances of the same event, since we ingested data twice.
+# Assert there are 2 instances of the same event, since we ingested data twice.
```

Comment:
Thanks for fixing that, not sure how I managed to commit that without the comment prefix 😓